tests/test_navbar.py

```diff
@@ -316,3 +318,93 @@ def assert_donate_page(driver, donate_page):
     assert 'support-cos' in driver.current_url
     assert meta_tag.get_attribute('property') == 'og:title'
     assert meta_tag.get_attribute('content') == 'Support COS'
+
+@markers.smoke_test
+@markers.core_functionality
+class TestCollectionsNavbarLoggedOut():
+    @pytest.fixture
+    def provider(self, driver):
+        return osf_api.get_provider(type='collections', provider_id='selenium')
+
+    @pytest.fixture()
+    def collectionsdiscover_page(self, driver, provider):
```

Comment:
Nitpick: can you add an underscore between `collections` and `discover`

Reply:
sure. will do